src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	public String STRING1 = "123456789";
```

Comment:
You must give meaningful variable names. It is not clear what you are storing in these variables.